projects/storefrontlib/cms-components/anonymous-consent-management/banner/anonymous-consent-management-banner.component.ts

```diff
@@ -25,7 +25,7 @@ export class AnonymousConsentManagementBannerComponent implements OnDestroy {
     this.hideBanner();
     const dialog = this.launchDialogService.openDialog(
       LAUNCH_CALLER.ANONYMOUS_CONSENT,
-      null,
+      undefined,
```

Comment:
why change the behaviour?

Reply:
the second param of `openDialog` is `ElementRef` or `undefined`

Reply:
Got it, thanks.